app/globals.css

```
@@ -450,23 +463,41 @@ button:focus-visible, a:focus-visible { outline: 2px solid var(--ink); outline-o
+.matter-shell:has(.material-address-layer[data-address-variant="structural"][data-material-address-painted]) .spatial-thought[data-selected="true"] .spatial-thought__label { background: transparent; box-shadow: none; text-shadow: none; }
+.matter-shell[data-material-address-owner="native"] .spatial-thought[data-selected="true"] .spatial-thought__label { background: transparent; box-shadow: none; text-shadow: none; }
+.matter-shell:has(.material-address-layer[data-address-variant="native"][data-material-address-painted])::selection,
+.matter-shell:has(.material-address-layer[data-address-variant="native"][data-material-address-painted]) ::selection { background: transparent; }
 .matter-shell[data-canvas-theme="dark"] .stretch-handle::after { background: color-mix(in srgb, currentColor 68%, transparent); box-shadow: 0 0 0 4px #202831; opacity: 1; animation-name: stretch-grip-arrive-dark; }
 .matter-shell[data-canvas-theme="dark"] .stretch-handle:hover::after,
 .matter-shell[data-canvas-theme="dark"] .stretch-handle:active::after,
 .matter-shell[data-canvas-theme="dark"] .stretch-handle[data-active="true"]::after { background: currentColor; }
-[data-stretch-handle="top"] .language-pocket { transform-origin: bottom center; }
-.elastic-preview[data-preview-mode="expand"] .language-pocket { opacity: 1; transform: scaleY(1); }
-.lasso-layer:has(.elastic-preview[data-preview-mode="expand"]) .lasso-selection-fragment { animation: none; opacity: 0; }
 .visually-hidden { position: absolute !important; width: 1px !important; height: 1px !important; padding: 0 !important; margin: -1px !important; overflow: hidden !important; clip: rect(0 0 0 0) !important; white-space: nowrap !important; border: 0 !important; }
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)